boilerplate/test/i18n.test.ts

```diff
@@ -43,8 +43,8 @@ function iterate(obj, stack, array) {
 describe("i18n", () => {
   test("There are no missing keys", (done) => {
     // Actual command output:
-    // grep "Tx=\"\S*\"\|tx=\"\S*\"\|translate(\"\S*\"" -ohr './app' | grep -o "\".*\""
-    const command = `grep "Tx=\\"\\S*\\"\\|tx=\\"\\S*\\"\\|translate(\\"\\S*\\"" -ohr './app' | grep -o "\\".*\\""`
+    // grep "[T\|t]x=[{]\?\"\S*\"[}]\?\|translate(\"\S*\"" -ohr './app' | grep -o "\".*\""
```

Comment:
Do you mind adding a line to the comment block above indicating that this regex now also matches on brackets?

Reply:
Absolutely! Thank YOU all for making awesome open source stuff! I updated the comment - how does that look?

Reply:
Perfect 👍